app/src/main/java/com/ericg/pagingmovies/composables/MoviesList.kt

```diff
@@ -0,0 +1,24 @@
+package com.ericg.pagingmovies.composables
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.paging.compose.collectAsLazyPagingItems
+import androidx.paging.compose.items
+import com.ericg.pagingmovies.viewmodel.MovieViewModel
+
+@Composable
+fun MoviesList() {
+
+    val viewModel: MovieViewModel = hiltViewModel()
+    val movieItems = viewModel.movies.value.collectAsLazyPagingItems()
+
+    LazyColumn() {
```

Comment:
This is a nice approach!
However, if you wish to decouple `viewModel` and `moviewItems` from `MovieList()`, you can pass them as arguments wherever you call `MovieList()`